spec/services/search/feed_content_spec.rb

```diff
@@ -88,6 +88,18 @@
         doc_ids = feed_docs.map { |t| t.dig("id") }
         expect(doc_ids).to include(article1.id, article2.id)
       end
+
+      it "bumps scores for words closer together" do
```

Comment:
@mstruve I checked out your branch and reverted `Search::QueryBuilders::FeedContent` back to its original form. However, this spec remains green, probably because the search phrase matches more of the overall document. It also stays green when I change the body text of the second article to "Ruby is cool I dont know maybe" though.

Here are my local changes:

```diff
diff --git a/app/services/search/query_builders/feed_content.rb b/app/services/search/query_builders/feed_content.rb
index 2d695800c..891619296 100644
--- a/app/services/search/query_builders/feed_content.rb
+++ b/app/services/search/query_builders/feed_content.rb
@@ -101,13 +101,14 @@ module Search
       def build_queries
         @body[:query] = { bool: {} }
         @body[:query][:bool][:filter] = filter_conditions if filter_keys_present?
-        return unless query_keys_present?
-
-        @body[:query][:bool][:must] = query_conditions
-        # Boost the score of queries that match these conditions but if they dont match any,
-        # minimum_should_match: 0, then that is OK
-        @body[:query][:bool][:should] = match_phrase_conditions
-        @body[:query][:bool][:minimum_should_match] = 0
+        @body[:query][:bool][:must] = query_conditions if query_keys_present?
+        # return unless query_keys_present?
+
+        # @body[:query][:bool][:must] = query_conditions
+        # # Boost the score of queries that match these conditions but if they dont match any,
+        # # minimum_should_match: 0, then that is OK
+        # @body[:query][:bool][:should] = match_phrase_conditions
+        # @body[:query][:bool][:minimum_should_match] = 0
       end

       def filter_conditions
diff --git a/spec/services/search/feed_content_spec.rb b/spec/services/search/feed_content_spec.rb
index a8d0e1ea3..192f4480e 100644
--- a/spec/services/search/feed_content_spec.rb
+++ b/spec/services/search/feed_content_spec.rb
@@ -91,7 +91,7 @@ RSpec.describe Search::FeedContent, type: :service do

       it "bumps scores for words closer together" do
         allow(article1).to receive(:body_text).and_return("Ruby I dont know maybe is cool")
-        allow(article2).to receive(:body_text).and_return("Ruby is cool")
+        allow(article2).to receive(:body_text).and_return("Ruby is cool I dont know maybe")
         index_documents([article1, article2])
         query_params = { size: 5, search_fields: "ruby is cool" }
```

Reply:
Good catch!

The problem was the assertion, we were looking that both articles were included rather than what their order was. I updated article 2 with the additional text and then set the array assertion to equal and now if you comment out my new code it fails.

Reply:
Ah, thanks @mstruve, that makes much more sense. 👍